Maple2.Server.Game/Packets/NpcControlPacket.cs

```diff
@@ -14,21 +14,21 @@ public static class NpcControlPacket {
     public const short ANI_JUMP_A = -2;
     public const short ANI_JUMP_B = -3;
 
-    public static ByteWriter Control(params FieldNpc[] npcs) {
+    public static ByteWriter Control(bool firstSend, params FieldNpc[] npcs) {
```

Comment:
I'm not understanding this bool. It's not firstSend seeing as you have it looping in FieldManager as true.
And is it right to be defaulting an `IdleSequenceId` if the `PlayingSequence` is null? Wouldn't there _always_ be a playing sequence? Idle would be considered a sequence, would it not?

Reply:
It's for the first time npc control packet sends to guarantee that a -1 for "current animation" isn't sent. Also right now the AnimationState is primarily used for scheduling with the states that are meant to do something. All of the other animations have some logic that upholds that they're being played at the right times & speed except for idle because no behavior depends on idle. I can change it to play idle if the last animation ended & another wasn;t scheduled right after though.